tests/cell_test.cc

```diff
@@ -1,3 +1,5 @@
+#include <cmath>
```

Comment:
Why do you need `cmath`?

Reply:
I forgot to remove since I used it to check something before